demos/python_demos/action_recognition/action_recognition_demo/steps.py

```diff
@@ -27,11 +27,11 @@
 from .queue import Signal
 
 
-def run_pipeline(video, encoder, decoder, render_fn, fps=30):
+def run_pipeline(video, encoder, decoder, render_fn, decoder_seq_size=16, fps=30):
     pipeline = AsyncPipeline()
     pipeline.add_step("Data", DataStep(video), parallel=False)
     pipeline.add_step("Encoder", EncoderStep(encoder), parallel=False)
-    pipeline.add_step("Decoder", DecoderStep(decoder), parallel=False)
+    pipeline.add_step("Decoder", DecoderStep(decoder, sequence_size=decoder_seq_size), parallel=False)
```

Comment:
I can understand the sequence size being configurable when you're using the dummy decoder, but does it make sense to override it when using a real decoder? IMO, the code should only allow the `--seq` option to be set when `-m_de` is unspecified, and otherwise derive the sequence size from the decoder's input shape.

Reply:
Implemented.